apps/backoffice-v2/src/common/types.ts

```diff
@@ -123,3 +123,21 @@ export type PolymorphicComponentPropsWithRef<
 > = PolymorphicComponentProps<TElement, TProps> & { ref?: PolymorphicRef<TElement> };
 
 // /PolymorphicComponentProps
+
+export const Severity = {
+  CRITICAL: 'critical',
+  HIGH: 'high',
+  MEDIUM: 'medium',
+  LOW: 'low',
+} as const;
+
+export const Severities = [
+  Severity.CRITICAL,
+  Severity.HIGH,
+  Severity.MEDIUM,
+  Severity.LOW,
+] as const satisfies ReadonlyArray<TObjectValues<typeof Severity>>;
+
+export type TSeverity = (typeof Severities)[number];
+
+export type TSeverities = typeof Severities;
```

Comment:
These values are used across multiple domains, just renamed and moved to common.